src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep.java

```diff
@@ -168,6 +168,7 @@ private void doStart() throws Exception {
     }
 
     private static final class Overrider extends EnvironmentExpander {
+        private Set<String> sensitiveVariables = new HashSet<>();
```

Comment:
```suggestion
```

Reply:
removed in 06428a0516322ded27dffb274fdcf04017ab944e